### PR TITLE
[local-kafka] Update strimzi kafka operator to 0.28.0

### PR DESCRIPTION
**Pull request by pjthepooh**
`strimzi-kafka-operator` 0.26.0 takes forever to start on M1 chip, and it seems that version doesn't have a tag for arm64 neither. 0.28.0 takes 4-5mins on my M1, which is still be slower than Intel, but is significantly faster than previous version. Also exposed image config in values.yaml for customized settings.

### CODE REVIEW

File: charts/local-kafka/Chart.yaml [hunk 1]
```diff
@@ -2,13 +2,13 @@ apiVersion: v2
...
-version: 0.2.1
+version: 0.3.0
 appVersion: latest
 maintainers:
   - name: diranged
     email: matt@nextdoor.com
 dependencies:
   - name: strimzi-kafka-operator
-    version: 0.26.0
+    version: 0.28.0
```
Comment (saansh45) [inline charts/local-kafka/Chart.yaml:12]: why not version 0.27.0?

Comment (pjthepooh) [reply charts/local-kafka/Chart.yaml:12]: no particular reason, mainly its the latest and i tried it without seeing things broken.

File: charts/local-kafka/Chart.yaml [hunk 2]
```diff
@@ -2,13 +2,13 @@ apiVersion: v2
 name: local-kafka
 description: Local Development spinup of Strimzi-managed Kafka
 type: application
-version: 0.2.1
+version: 0.3.0
```
[the remark this answers is not in the document]
Comment (pjthepooh) [reply charts/local-kafka/Chart.yaml:5]: thinking just to be consistent with the minor version bump in the dependency.

File: charts/local-kafka/values.yaml [hunk 3]
```diff
@@ -35,6 +35,15 @@ listeners:
...
 strimzi-kafka-operator:
+
+  # These overwrite default Cluster Operator image settings.
+  # E.g. one may need to set image.tag "0.28.0-arm64" explicitly for ARM64 arch
+  image:
+    registry: ""
+    repository: ""
+    name: operator
+    tag: ""
+
```
[the remark this answers is not in the document]
Comment (pjthepooh) [reply charts/local-kafka/values.yaml:46]: sure will od.

**Review by saansh45 — APPROVED** [review]
👍

**Pull request merged**